tool_scoring.R

```diff
@@ -0,0 +1,458 @@
+library(synapser)
```

Comment:
I would also recommend adding a step to create a new version of the tables you are writing to at the end of the run, that way we can revert if we need to (particularly if something unexpected happens in the first few weeks of using this)